src/app/index.ts

```diff
@@ -6,21 +6,21 @@ import * as _ from 'lodash';
 import * as appInsights from 'applicationinsights';
 import * as chalk from 'chalk';
 import * as fs from 'fs';
+import { helperMethods } from './helpers/helperMethods';
+import { modifyManifestFile } from 'office-addin-manifest';
 import * as path from "path";
+import projectsJsonData from './config/projectsJsonData';
 import * as uuid from 'uuid/v4';
 import * as yosay from 'yosay';
 import * as yo from 'yeoman-generator';
-import generateStarterCode from './config/starterCode';
-import projectsJsonData from './config/projectsJsonData';
-import { helperMethods } from './helpers/helperMethods';
-import { modifyManifestFile } from 'office-addin-manifest';
 
-let insight = appInsights.getClient('1ced6a2f-b3b2-4da5-a1b8-746512fbc840');
-let git = require("simple-git");
+const insight = appInsights.getClient('1ced6a2f-b3b2-4da5-a1b8-746512fbc840');
+const git = require("simple-git");
```

Comment:
We need to remove the git requirement and download the zip file instead.

Reply:
@akrantz Downloading the zip is not as trivial as I hoped. While downloading and extracting the zip is straighforward, the extracted project is created in a subfolder under the project directory, and this will require some clever moving around and deletion of files and folders following the download/unzip.  I am going to open an issue to do this work and will cite the working branch that has these changes

Reply:
new issue opened: https://github.com/OfficeDev/generator-office/issues/431